app/assets/stylesheets/v3/layout_components/responsive-table.scss

```diff
@@ -198,3 +200,80 @@ $use-respond-mixins: false !default;
     margin-bottom: 0;
   }
 }
+
+// This is the new table css as of Nov 2020. Please use this going forward.
+
+.responsive-table {
+  border-style: none;
+  border-collapse: collapse;
+  width: 100%;
+  table-layout: fixed;
+
+  thead,
+  tbody {
+    border: none;
+  }
+
+  th,
+  td {
+    padding: $base-padding;
+    text-align: left;
+    border-right: 1px solid $bg-grey-light;
+
+    &:last-child {
+      border-style: none;
+    }
+  }
+
+  th {
+    background-color: $bg-grey-lighter;
+  }
+
+  tr {
+    background-color: $white;
+  }
+
+  tbody tr:nth-child(even) {
+    background-color: $bg-grey-lightest;
+  }
+
+  @include breakpoint(small only) {
```

Comment:
instead of having all of the mobile styles in one big block, can we have separate mobile includes as part of each selector above? i.e.:

```
tr {
  background-color: $white;

    @include breakpoint(small only) {
        display: block;
        padding: $base-padding;
        font-size: $base-font-size;
        border-top: 1px solid $bg-grey-light !important;

        &:last-child {
            border-bottom: 1px solid $bg-grey-light !important;
        }
  } 

```

Reply:
👍  good call!